src/demo/example-data/piechart.json

```diff
@@ -0,0 +1,8 @@
+{
+    "pies": [ {"x": 433600, "y": 6477600, "R": 99, "fractions": [21, 15, 45, 32]},
+              {"x": 434556, "y": 6478951, "R": 99, "fractions": [18, 11, 15]},
+              {"x": 437086, "y": 6477198, "R": 99, "fractions": [12, 28, 22, 9]}
+            ],
+
+     "colors": [[255, 0, 0], [0, 255, 0], [0, 0, 255], [255, 255, 0], [0, 255, 255], [255, 0, 255]]       
```

Comment:
I guess this is already discussed, but instead of just colors, this could include labels as well, and maybe any kind of data. 
```
"fractionAttributes"= [{"color": [255, 0, 0], "label":"blabla"}, {"color...
```
Also, the mapping between the fractions and the fraction color is based on the order, I guess? So it needs to work in cases where a fraction is 0, to make it possible to have cases like this, for example: 3 slices and 5 colors, where slice 1 maps to first color, slice 2 to the third color and slice 3 to the last color. You could also change the format to specify explicitly which fraction attribute to use for which fraction, up to you.